iomgr/lib/dbengine/Database_Common.cpp

```diff
@@ -98,6 +98,23 @@ std::vector<TableRecord> Database::getTableRecordsOrderedByName(bool includeSyst
     return result;
 }
 
+std::vector<ColumnRecord> Database::getColumnsRecordsOrderedByName(std::uint32_t tableId) const
```

Comment:
This is incorrect way to get table columns. It will, however, work, but until adding/removing columns are implemented. Then it will produce both old & new columns. The correct way to do this is a bit more sophisticated.

Reply:
So please remove this function and rewrite logic as described in the other comment.